### PR TITLE
[NUI][ImageView] Fix ImageView guide

### PR DESCRIPTION
**Pull request by pkalota**
### Change Description ###

- Provide one link to ImageView latest api at the start, rest is changed to `ImageView`
- Fix a little 'Creating an ImageView' section
- Add simple code at 'Manipulating an ImageView' section
- Add information about ImageView event
- Add screenshots from code snippets
- Add link to example in related information

### CODE REVIEW

File: docs/application/dotnet/guides/nui/imageview.md [hunk 1]
```diff
@@ -1,39 +1,101 @@
+---
+keyword: image, ImageView, subarea, premultipliedalpha, PixelArea, ResourceReady, Orientation
+---
+
 # ImageView
 
-An image view is a NUI control that displays an image. It is implemented through the `Tizen.NUI.BaseComponents.ImageView` class.
+An image view is a NUI control that displays an image. It is implemented through the [ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/imageview.md:7]: An `ImageView `is a NUI control that displays an image. It is implemented through the [ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class:

File: docs/application/dotnet/guides/nui/imageview.md [hunk 2]
```diff
@@ -1,39 +1,101 @@
...
+
+2.  To change the image path after the instance has been created, use the `SetImage()` method of the `ImageView` class:
+
+    ```csharp
     imageView = new ImageView();
-    imageView.ResourceUrl = DirectoryInfo.Resource + "images/gallery-3.jpg";
+    imageView.SetImage(DirectoryInfo.Resource + "your-image.jpg");
+    Window.Instance.Add(imageView);
     ```
+## Manipulating an Image View
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/imageview.md:46]: ## Manipulate an ImageView

File: docs/application/dotnet/guides/nui/imageview.md [hunk 3]
```diff
@@ -1,39 +1,101 @@
+---
+keyword: image, ImageView, subarea, premultipliedalpha, PixelArea, ResourceReady, Orientation
+---
+
 # ImageView
 
-An image view is a NUI control that displays an image. It is implemented through the `Tizen.NUI.BaseComponents.ImageView` class.
+An `ImageView` is a NUI control that displays an image. It is implemented through the [ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/imageview.md:7]: I am sorry, by mistake I added a colon at the end of the sentence, kindly change it back to a period.

An `ImageView` is a NUI control that displays an image. It is implemented through the [ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class.

File: docs/application/dotnet/guides/nui/imageview.md [hunk 4]
```diff
@@ -1,39 +1,101 @@
...
+
+2.  Use `PixelArea` property to apply the zoom effect:
+
+    ```csharp
+    imageView.PixelArea = new RelativeVector4(0.25f, 0.25f, 0.5f, 0.5f);
     ```
 
-<a name="properties"></a>
-## Image View Properties
+3.  To rotate image by 45 degrees, set the `Orientation` property as follow:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/imageview.md:69]: 3.  To rotate the image by 45 degrees, set the `Orientation` property as follows:

File: docs/application/dotnet/guides/nui/imageview.md (hunk 2 above)
Comment (pkalota) [reply docs/application/dotnet/guides/nui/imageview.md:46]: Sorry I've changed figure name instead of section name before. It will be fixed in next push.

**Review by mijin85cho — COMMENTED** [review]
Hello, @annie-samsung 
I skim through the stg page and found below points. Would you please also check these, too?

File: docs/application/dotnet/guides/nui/imageview.md [hunk 5]
```diff
@@ -1,53 +1,117 @@
...
 | `PixelArea`          | `Vector4`   | Image subarea defined with relative area  values: x coordinate for the top-left corner, y coordinate for the top-left corner, width, and height. To set the subarea as the whole image area, use `[0.0, 0.0, 1.0, 1.0]`.<br>For example, on a 200 x 200 pixel image, the  value `[0.25, 0.5, 0.5, 0.5]` represents a subarea of   that image with the  following coordinates:<br><br>-   Top left: 50, 100<br>-   Top right: 150, 100<br>-   Bottom left: 50, 200<br>-   Bottom right: 150, 200        |
 | `Border`             | `Rectangle` | Image border, specified  in the following order:  left, right, bottom, and  top. For N-Patch images only.         |
 | `BorderOnly`         | `bool`      | Whether to draw only the borders. For N-Patch  images only.  |
-| `SynchronousLoading` | `bool`      | Whether the image is synchronous. For N-Patch images only.     |
+| `SynchronosLoading`  | `bool`      | Whether the image is synchronos. For N-Patch images only.     |
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/imageview.md:110]: I think you could have retained the correct spelling here...

Whether the image is synchronous. For N-Patch images only.

**Comment by hyunjushin** [comment]
Please specify all image formats supported by NUI.

**Review by annie-samsung — APPROVED** [review]
Approving

**Pull request merged**